console/modules/hf-process.js

```diff
@@ -52,22 +95,77 @@ Process.prototype = extend(Process.prototype, {
             return;
         }
         console.log("Starting " + this.command);
+
+        var logDirectoryCreated = false;
+        try {
+            fs.mkdirSync('logs');
```

Comment:
This is likely fine for now, I'm guessing we'll want to put logs in a common directory per OS so they are readable separate from console.